webapp/publisher/snaps/builds.py

```diff
@@ -35,13 +35,21 @@ class LaunchpadStoreUploadState(Enum):
     UPLOADED = "Uploaded"
 
 
+def get_github_repository(github_repo_url):
+    """ Returns a string with the GitHub owner
+        and the repository name separated by an
+        slash: <Owner>/<Repository name>
+    """
+    return github_repo_url.split("https://github.com/")[-1]
```

Comment:
This really seems so trivial that creating its own function just obfuscates code. Putting this logic inline would make the original code literally no larger.

I would just do this inline:

``` python3
# Remove hostname & split owner/repo
owner, repo = github_repo_url[19:].split('/')
```

Reply:
Okay, I like this, I guess I did it with a function because in the next PR I need to repeat this operation.

Reply:
Yeah, I assumed you must be doing it in more than one place. And while we should be DRY, I think you really get diminishing returns on code re-use when it's just one or two lines. I personally think just repeating this logic in 2 or 3 places so that you can make the code more transparent is more important than DRYness in this case.